python_benchmarks/global_params.py

```diff
@@ -2,24 +2,28 @@
 from typing import Union, List
 from nvfuser import Datatype
 
+
 # Utility function to generate input sizes for benchmarks
 def generate_input_sizes(dims: Union[int, List] = 2) -> List[tuple]:
```

Comment:
Let's specify the element type if possible, e.g., `List[int]` and `List[Tuple[int]]`.

Reply:
The input list can be a mix of various types of inputs (Tensor, int, bool, float) so for simplicity I left it at `List` for now.